pkgs/top-level/python-packages.nix

```diff
@@ -21125,6 +21125,8 @@ EOF
   wsproto = callPackage ../development/python-modules/wsproto { };
 
   h11 = callPackage ../development/python-modules/h11 { };
+
+  Rtree = callPackage ../development/python-modules/Rtree { inherit (pkgs) libspatialindex; };
```

Comment:
s/Rtree/rtree/g

Reply:
Really? The library is actually called `Rtree` and I see plenty of other packages whose names reflect their actual capitalization.

Reply:
I prefer lowercase, but I think it's up to you.
However, plaese try to stick with the alphabetical ordering of `python-packages.nix`.